docs/docs/tutorial_sentinel2_ndvi.md

```diff
@@ -0,0 +1,653 @@
+Sentinel-2 NDVI computation
+===========================
+
+The following asynchronous API call computes the NDVI of the Sentinel-2
+scene
+**S2A\_MSIL1C\_20161206T030112\_N0204\_R032\_T50RKR\_20161206T030749**:
+
+```bash
+ curl ${AUTH} -X POST "${ACTINIA_URL}/sentinel2_process/ndvi/S2A_MSIL1C_20161206T030112_N0204_R032_T50RKR_20161206T030749"
```

Comment:
does not work:

curl ${AUTH} -X GET "https://actinia.mundialis.de/api/v1/resources/demouser/resource_id-39a95083-aa89-4075-a5f2-1e70956ea78a" | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2127  100  2127    0     0   1217      0  0:00:01  0:00:01 --:--:--  1217
{
  "accept_datetime": "2021-10-05 13:49:22.620107",
  "accept_timestamp": 1633441762.6201046,
  "api_info": {
    "endpoint": "asyncephemeralsentinel2processingresource",
    "method": "POST",
    "path": "/api/v1/sentinel2_process/ndvi/S2A_MSIL1C_20161206T030112_N0204_R032_T50RKR_20161206T030749",
    "request_url": "http://actinia.mundialis.de/api/v1/sentinel2_process/ndvi/S2A_MSIL1C_20161206T030112_N0204_R032_T50RKR_20161206T030749"
  },
  "datetime": "2021-10-05 13:49:22.781323",
  "exception": {
    "message": "AsyncProcessError:  Error in querying Sentinel-2 product <S2A_MSIL1C_20161206T030112_N0204_R032_T50RKR_20161206T030749> in Google BigQuery Sentinel-2 database. Error: GoogleCloudAPIError:  An error occurred while fetching Sentinel-2 download URL's. Error message: File /etc/GCS_credentials.json was not found.",
    "traceback": [
      "  File \"/usr/lib/python3.8/site-packages/actinia_core/rest/ephemeral_processing.py\", line 1747, in run\n    self._execute()\n",
      "  File \"/usr/lib/python3.8/site-packages/actinia_satellite_plugin/ephemeral_sentinel2_ndvi_processor.py\", line 968, in _execute\n    self._prepare_sentinel2_download()\n",
      "  File \"/usr/lib/python3.8/site-packages/actinia_satellite_plugin/ephemeral_sentinel2_ndvi_processor.py\", line 788, in _prepare_sentinel2_download\n    raise AsyncProcessError(\"Error in querying Sentinel-2 product <%s> \"\n"
    ],
    "type": "<class 'actinia_core.core.common.exceptions.AsyncProcessError'>"
  },
  "http_code": 400,
  "message": "AsyncProcessError:  Error in querying Sentinel-2 product <S2A_MSIL1C_20161206T030112_N0204_R032_T50RKR_20161206T030749> in Google BigQuery Sentinel-2 database. Error: GoogleCloudAPIError:  An error occurred while fetching Sentinel-2 download URL's. Error message: File /etc/GCS_credentials.json was not found.",
  "process_chain_list": [],
  "process_log": [],
  "progress": {
    "num_of_steps": 0,
    "step": 0
  },
  "resource_id": "resource_id-39a95083-aa89-4075-a5f2-1e70956ea78a",
  "status": "error",
  "time_delta": 0.16125011444091797,
  "timestamp": 1633441762.7812667,
  "urls": {
    "resources": [],
    "status": "https://actinia.mundialis.de/api/v1/resources/demouser/resource_id-39a95083-aa89-4075-a5f2-1e70956ea78a"
  },
  "user_id": "demouser"
}